src/vs/editor/browser/editorBrowser.ts

```diff
@@ -1227,6 +1227,10 @@ export function getCodeEditor(thing: unknown): ICodeEditor | null {
 		return thing.getModifiedEditor();
 	}
 
+	if (isCompositeEditor(thing) && isCodeEditor(thing.activeCodeEditor)) {
+		return thing.activeCodeEditor;
+	}
```

Comment:
This function is popular and here in the following files


* src/vs/editor/contrib/quickAccess/browser/editorNavigationQuickAccess.ts
* src/vs/editor/contrib/quickAccess/browser/gotoLineQuickAccess.ts
* src/vs/workbench/browser/parts/editor/editorStatus.ts
* src/vs/workbench/contrib/format/browser/formatActionsMultiple.ts
* src/vs/workbench/contrib/languageDetection/browser/languageDetection.contribution.ts
* src/vs/workbench/contrib/languageStatus/browser/languageStatus.contribution.ts
* src/vs/workbench/contrib/search/browser/searchView.ts
* src/vs/workbench/contrib/snippets/browser/commands/fileTemplateSnippets.ts
* src/vs/workbench/services/editor/browser/codeEditorService.ts
* src/vs/workbench/services/preferences/browser/preferencesService.ts

Are we sure that we don't change their behaviour? Did this get tested? Given endgame nears I wouldn't do such a global change and keep things local (as before)

Reply:
I'm fine with holding this until after endgame if you'd like for further testing. I think this is a safe change, I have not tested all those cases though and have relied on just quick sanity testing along with our unit and integration tests. The `codeEditorService` has this exact implementation for `getActiveCodeEditor`

https://github.com/microsoft/vscode/blob/4bf364de22d1c098fc5754f475a97480c1ef92fd/src/vs/workbench/services/editor/browser/codeEditorService.ts#L32-L48

Reply:
Let's take chances ;-)